DNN Platform/Website/admin/Skins/Logo.ascx.cs

```diff
@@ -58,6 +64,40 @@ protected override void OnLoad(EventArgs e)
                         {
                             this.imgLogo.ImageUrl = imageUrl;
                             logoVisible = true;
+                        }
+
+                        if (this.InjectSVG.GetValueOrDefault() == true)
+                        {
+                            logoVisible = false; // hide the <img>, we are going to inject an <svg> tag instead.
+
+                            hypLogo.CssClass = this.CssClass; // put the custom CSS class on the hyperlink instead
+
+                            string svg = string.Empty;

```

Comment:
This declaration isn't needed, please remove this if we can to keep string instantiation reduced

Reply:
Is there a better way to do this? 

I am setting the value if the string inside an if block and referencing the value later from outside of the if block.

Reply:
Just use the cacheSvg value from below, no need for two strings